lib/kitchen/provisioner/chef_base.rb

```diff
@@ -49,6 +49,7 @@ class ChefBase < Base
       default_config :attributes, {}
       default_config :config_path, nil
       default_config :log_file, nil
+      default_config :log_level, 'auto'
```

Comment:
It's passed to the command line, so will get converted to a string at that point anyway, but I can make it a symbol for consistancy